types/moment-range/index.d.ts

```diff
@@ -1,104 +1,62 @@
+    isSame(other: DateRange): boolean;
 
-        isSame(other: DateRange): boolean;
+    diff(unit?: unitOfTime.Diff, rounded?: boolean): number;
 
-        diff(unit?: unitOfTime.Diff, rounded?: boolean): number;
+    toDate(): Date[];
 
-        toDate(): Date[];
+    toString(): string;
 
-        toString(): string;
+    valueOf(): number;
 
-        valueOf(): number;
+    center(): Moment;
 
-        center(): Moment;
+    clone(): DateRange;
 
-        clone(): DateRange;
+    isEqual(other: DateRange): boolean;
 
-        isEqual(other: DateRange): boolean;
+    adjacent(other: DateRange): boolean;
 
-        adjacent(other: DateRange): boolean;
+    duration(unit?: unitOfTime.Diff, precise?: boolean): number;
 
-        duration(unit?: unitOfTime.Diff, precise?: boolean): number;
+    reverseBy(interval: unitOfTime.Diff, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
 
-        reverseBy(interval: unitOfTime.Diff, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
-
-        reverseByRange(interval: DateRange, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
-    }
-
-    interface Range {
-        start: Moment;
-        end: Moment;
-
-        contains(other: DateRange | Moment | Date, options?: { exclusive?: boolean }): boolean;
-
-        overlaps(range: DateRange, options?: { adjacent?: boolean }): boolean;
-
-        intersect(other: DateRange): DateRange;
-
-        add(other: DateRange): DateRange;
-
-        subtract(other: DateRange): DateRange[];
-
-        by(interval: unitOfTime.Diff, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
-
-        byRange(interval: DateRange, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
-
-        isSame(other: DateRange): boolean;
-
-        diff(unit?: unitOfTime.Diff, rounded?: boolean): number;
-
-        toDate(): Date[];
-
-        toString(): string;
-
-        valueOf(): number;
-
-        center(): Moment;
-
-        clone(): DateRange;
-
-        isEqual(other: DateRange): boolean;
-
-        adjacent(other: DateRange): boolean;
-
-        duration(unit?: unitOfTime.Diff, precise?: boolean): number;
-
-        reverseBy(interval: unitOfTime.Diff, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
-
-        reverseByRange(interval: DateRange, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
-    }
-
-    function extendMoment(moment: any): any;
+    reverseByRange(interval: DateRange, options?: { exclusive?: boolean, step?: number }): Iterable<Moment>;
+}
 
-    function range(range: string | Date[] | Moment[]): Range;
-    function range(start: Date | Moment, end: Date | Moment): Range;
+export interface MomentRangeExtends {
+    range(range: string | Date[] | Moment[]): DateRange;
+    range(start: Date | Moment, end: Date | Moment): DateRange;
 }
```

Comment:
It makes sense to include within function here, when extending Moment type.
Thanks!